spec/spec_helper_local.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+dir = __dir__
+
+require 'augeas_spec'
+
+# augeasproviders: setting $LOAD_PATH to work around broken type autoloading'
+$LOAD_PATH.unshift(
+  dir,
+  File.join(dir, 'fixtures/modules/augeasproviders_core/spec/lib'),
```

Comment:
I wonder if this is needed. The tests pass even if this is removed. It also has a `require 'augeas_spec'` before this unshift but `augeas_spec` is inside this directory.